examples/.config/model_params_pytorch_3x.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pytorch": {
```

Comment:
The 3x Pytorch examples should use the same config file, so the 3x llm examples' config should also added here. 
The model list is: llama2_7b_gptq_int4,llama2_7b_gptq_int4_dq_bnb,llama2_7b_gptq_int4_dq_ggml,gpt_j_woq_rtn_int4,gpt_j_woq_rtn_int4_dq_bnb,gpt_j_woq_rtn_int4_dq_ggml,gpt_j_woq_gptq_int4,gpt_j_woq_gptq_int4_dq_bnb,gpt_j_woq_gptq_int4_dq_ggml,opt_125m_woq_gptq_int4,opt_125m_woq_gptq_int4_dq_bnb,opt_125m_woq_gptq_int4_dq_ggml  
opt_125m_ipex,opt_125m_ipex_sq,llama2_7b_ipex,llama2_7b_ipex_sq,gpt_j_ipex,gpt_j_ipex_sq

Reply:
We will rename the folder from `llm` to `weight_only` and  `smooth_quant`, so it won't be included in this PR.  
@Kaihui-intel @violetch24 please follow Suyue's requirement.